SPECS/lynx/lynx-proxy-empty-string.patch

```diff
@@ -1,15 +1,26 @@
-Index: lynx2.8.9rel.1/WWW/Library/Implementation/HTAccess.c
-===================================================================
---- lynx2.8.9rel.1/WWW/Library/Implementation/HTAccess.c	2014-02-22 20:28:32.000000000 +0000
-+++ lynx2.8.9rel.1/WWW/Library/Implementation/HTAccess.c	2014-02-22 20:28:45.000000000 +0000
-@@ -513,6 +513,9 @@ static int get_physical(const char *addr
+From af24900d0964ed6c1cc8688ba8d2f8ccc75b239b Mon Sep 17 00:00:00 2001
+From: Thomas Crain <thcrain@microsoft.com>
+Date: Tue, 14 Dec 2021 11:52:12 -0800
+Subject: [PATCH 3/3] empty proxy patch
+
+---
+ WWW/Library/Implementation/HTAccess.c | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/WWW/Library/Implementation/HTAccess.c b/WWW/Library/Implementation/HTAccess.c
+index a79a8a2..b8b0df3 100644
+--- a/WWW/Library/Implementation/HTAccess.c
++++ b/WWW/Library/Implementation/HTAccess.c
+@@ -513,6 +513,9 @@ static int get_physical(const char *addr,
  	else
  	    sprintf(gateway_parameter, "%s_proxy", acc_method);
  	proxy = LYGetEnv(gateway_parameter);
 +	if (proxy)
-+	    if (*proxy == '\0')
```

Comment:
Is the only difference here whitespace?

Reply:
Theoretically yes, but I was having trouble applying the original patch against the source, so I redid this one as well.